server/src/test/java/org/cloudfoundry/identity/uaa/oauth/UaaTokenStoreTests.java

```diff
@@ -350,7 +350,7 @@ void testCountingTheExecutedSqlDeleteStatements() throws SQLException {
         // When
         for (int i = 0; i < 10; i++) {
             try {
-                store.performExpirationClean();
+                store.performExpirationCleanIfEnoughTimeHasElapsed();
```

Comment:
Are you confident that there is sufficient test coverage for the parallel use case?

Some questionable things in this test - 
 - Doesn't seem to call the method concurrently.
 - With "atMost(1)" it may be non-deterministic. Why not expect exactly 1 call?
 - Could it really run successfully for close to 5 minutes?